### PR TITLE
Goring qoni

### PR DESCRIPTION
**Pull request by udsardor**

### CODE REVIEW

File: js/non-home.js [hunk 1]
```diff
@@ -1 +1,7 @@
-ichina code yozib qosh keyingi safar
+window.addEventListener("DOMContentLoaded", () => {
+    function sayHello(name){
+        console.log(name + "Salom");
+    }
+
+    sayHello("sardor")
+})
```
Comment (nurullo0) [inline js/non-home.js:7]: ajoyib code @udsardor  menga yoqdi

**Review by nurullo0 — COMMENTED** [review]
ishladi task bajarilibdi

**Comment by nurullo0** [comment]
zo'r

**Comment by udsardor** [comment]
vaoo 
zooorrr

**Comment by nurullo0** [comment]
choti nachundir qoshib bolmin duribdi

**Review by nurullo0 — COMMENTED** [review]
a

**Pull request merged**